### PR TITLE
streamline cn commands

### PR DESCRIPTION
**Pull request by hsdk123**
## Reason for change

- make sure options align with command names, etc.

### CODE REVIEW

File: Data/Commands.xml [hunk 1]
```diff
@@ -60,7 +60,7 @@
       </option>
       <option name="on_camera">
         <val lang="jap" value="カメラ付着"/>
-        <val lang="zh_cn" value="视角附着"/>
+        <val lang="zh_cn" value="绑定视角"/>
```
Comment (hsdk123) [inline Data/Commands.xml:63]: Reason: keep consistent with `自动绑定视角`

File: Data/Commands.xml [hunk 2]
```diff
@@ -228,7 +224,7 @@
       </option>
       <option name="startAtTarget">
         <val lang="jap" value="対象位置から"/>
-        <val lang="zh_cn" value="从目标位置开始"/>
+        <val lang="zh_cn" value="从目标位置"/>
```
Comment (hsdk123) [inline Data/Commands.xml:227]: Reason: keep consistent with `视角跟随`:
```
1. 视角跟随 [目标名] (x) (y) (r) (相对x) (相对y) (相对r) (大约完成时间:毫秒) (从目标位置)
```

File: Data/Commands.xml [hunk 3]
```diff
@@ -3614,7 +3610,7 @@
                    details="例如：自动绑定视角 true
                    &#xA;
                    &#xA;如果为true，
-                   &#xA;在当前脚本中，执行命令后的所有个体将自动绑定到视角。
+                   &#xA;在当前脚本中，执行命令后的所有个体将自动 [绑定视角]。
```
Comment (hsdk123) [inline Data/Commands.xml:3613]: Reason: make this is adding the `绑定视角` option more clearer

File: Data/Commands.xml [hunk 4]
```diff
@@ -5926,7 +5922,7 @@
                    details="例如：基于视角的对象绘制 true
                    &#xA;
                    &#xA;true)
-                   &#xA;根据与视角的距离顺序绘制对象。
+                   &#xA;根据与视角的距离(z)顺序绘制对象。
```
Comment (hsdk123) [inline Data/Commands.xml:5925]: Reason: make distance is calculated based on z more clearer.

File: Data/Commands.xml [hunk 5]
```diff
@@ -7125,16 +7121,16 @@
                    &#xA;if [wait ]"
                    reference="https://wikiwiki.jp/lightvn/commands/script#continueRead">
       </explanation>
-      <explanation lang="zh_cn" name="继续读取"
-                   syntax="继续读取 (Light.vn命令)"
-                   details="例如：继续读取
+      <explanation lang="zh_cn" name="继续"
```
Comment (hsdk123) [inline Data/Commands.xml:7124]: Reason: Make consistent with
```
&#xA;- 等待 禁止继续
&#xA;禁止通过“继续”命令重新启动脚本读取。
&#xA; 结果：当前脚本只能通过“跳转”重新启动读取。
&#xA;例如：等待 禁止继续
```

File: Data/Commands.xml [hunk 6]
```diff
@@ -7268,7 +7264,7 @@
                    &#xA;等待“对话框结果”命令。
                    &#xA;例如：等待 消息框选择
                    &#xA;
-                   &#xA;- 等待 视频结束
+                   &#xA;- 等待 视频结束 (允许用户终止)
```
Comment (hsdk123) [inline Data/Commands.xml:7267]: Reason: `允许用户终止` option forgotten

File: Data/Commands.xml [hunk 7]
```diff
@@ -7774,7 +7778,7 @@
                    &#xA;移除所有在堆栈底部脚本之上的脚本，并继续读取底部脚本。
                    &#xA;（如果当前脚本是第一个脚本，则不执行任何操作。）
                    &#xA;
-                   &#xA;注意：屏幕区域也将恢复到没有设置[freeOnReturnToMain]的最上层屏幕区域。
+                   &#xA;注意：屏幕区域也将恢复到没有设置[返回主界面时释放]的最上层屏幕区域。
```
Comment (hsdk123) [inline Data/Commands.xml:7781]: Reason: use the `返回主界面时释放` defined in 
```
<option name="freeOnReturnToMain">
        <val lang="jap" value="原点回帰時解除"/>
        <val lang="zh_cn" value="返回主界面时释放"/>
</option>
```

File: Data/Commands.xml [hunk 8]
```diff
@@ -228,19 +224,19 @@
...
+        <val lang="zh_cn" value="从目标位置"/>
       </option>
       <option name="preventEntry">
         <val lang="jap" value="侵入禁止"/>
         <val lang="zh_cn" value="禁止进入"/>
       </option>
       <option name="stopOnCollisionX">
         <val lang="jap" value="X衝突時初期化"/>
-        <val lang="zh_cn" value="X轴碰撞时重置"/>
+        <val lang="zh_cn" value="X碰撞时重置"/>
```
Comment (hsdk123) [inline Data/Commands.xml:235]: Reason: streamline with below
```
syntax="加速度x [目标名 (可多个)] [数值] (所需时间:毫秒) (加速度: a0~a18) (X碰撞时重置) (Y碰撞时重置) (画面区域范围)"
details="例如：加速度x albert 30 300
 ```

File: Data/Commands.xml [hunk 9]
```diff
@@ -1783,15 +1786,15 @@
...
-                   syntax="trackbar [name] [x] [y] [layer] [width] [variable name] [left val] [right val]
+                   syntax="trackbar [name] [x] [y] [layer] [horizontal/vertical] [length] [variable name] [left val] [right val]
                    &#xA;(ignoreTrans) (camera_option) (command to trigger on value change complete)"
                    details="ex. trackbar trackbar 370 246 105 530 vn_sysMasterVolume 0 100 sys_trackball.png  
                    &#xA;creates a trackbar which controls the variable 'vn_sysMasterVolume'
                    &#xA;"
                    reference="https://wikiwiki.jp/lightvn/commands/UI#trackbar">
       </explanation>
       <explanation lang="zh_cn" name="轨迹条"
-                   syntax="轨迹条 [名称] (轨迹球未触摸素材) (轨迹球触摸素材) (轨迹球点击素材) [x] [y] [r] [水平/垂直] [宽度] [变量名称] [左侧值] [右侧值]"
```
Comment (hsdk123) [inline Data/Commands.xml:1794]: Reason: Unify with
```
      <option name="vertical">
        <val lang="jap" value="縦"/>
        <val lang="zh_cn" value="竖"/>
      </option>
      <option name="horizontal">
        <val lang="jap" value="横"/>
        <val lang="zh_cn" value="横"/>
      </option>
```

File: Data/Commands.xml [hunk 10]
```diff
@@ -8497,15 +8521,15 @@
                    reference="https://wikiwiki.jp/lightvn/commands/media#voice">
       </explanation>
       <explanation lang="zh_cn" name="语音"
-                   syntax="语音 [名字] [文件名] (音量变量) (开/关变量：true/false) (禁止后台记录)"
+                   syntax="语音 [名字] [文件名] (音量变量) (开/关变量：true/false) (禁止添加到日志)"
```
Comment (hsdk123) [inline Data/Commands.xml:8524]: Reason: make consistent with
```
<option name="noBacklogAddition">
    <val lang="jap" value="バックログ登録禁止"/>
    <val lang="zh_cn" value="禁止添加到日志"/>
  </option>
```

File: Data/Commands.xml [hunk 11]
```diff
@@ -8726,15 +8750,15 @@
...
+                   syntax="movie [name] [filename] [x] [y] [laye(r)] (repeat) (camera_option)"
                    details="ex. movie movie1 test.mp4 0 0 70  
                    &#xA;  
                    &#xA;Suffix: '0' will start play at alpha + volume: 0.   
                    &#xA; (ex. movie0 movie1 test.mp4 0 0 70)"
                    reference="https://wikiwiki.jp/lightvn/commands/media#movie">
       </explanation>
       <explanation lang="zh_cn" name="视频"
-                   syntax="视频 [名字] [文件名] [x] [y] [r] (重复) (摄像头选项)"
+                   syntax="视频 [名字] [文件名] [x] [y] [r] (重复) (视角选项)"
```
Comment (hsdk123) [inline Data/Commands.xml:8761]: Make consistent with other locations that use `视角选项`

**Comment by hsdk123** [comment]
@xiakkto if you could take a quick check of the above, would be great!

**Comment by xiakkto** [comment]
> @xiakkto if you could take a quick check of the above, and let me know if it's fine, would be great!

Everything seems fine. I'm also working on tweaking those commands that don't quite sound right. I'll send in another pull request in a few days. Thanks!

**Pull request merged**